#global testerror "Err0026"
func toto() => 0

const A = 10 + toto()